mobile/src/main/java/ch/epfl/sdp/mobile/ui/prepare_game/PrepareGameDialog.kt

```diff
@@ -0,0 +1,68 @@
+package ch.epfl.sdp.mobile.ui.prepare_game
+
+import androidx.compose.animation.animateContentSize
+import androidx.compose.animation.core.animateDpAsState
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.zIndex
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.social.Person
+
+@Composable
+fun <P : Person> PrepareGameDialog(
+    state: PrepareGameScreenState<P>,
+    modifier: Modifier = Modifier
+) {
```

Comment:
Missing some documentation

Reply:
Fixed in [3b6d9d6](https://github.com/epfl-SDP/android/pull/200/commits/3b6d9d609f7b69efe8b9d29e5165b1e8cd0b7a60)